### DISCUSSION TITLE
access to alarm properties

### DISCUSSION THREAD

**Discussion by GabrielTec251** [SCADA-LTS/Scada-LTS#2630] [Ideas]
I would like a way to get alarm properties, such as
*ALARM ACKNOWLEDGED
*ACTIVE ALARM
* INACTIVE ALARM

Reason, I would like to use it in graphic logics where for example:

In case of failure, the component must flash yellow,
 when the alarm is acknowledged but still in fault, the component is in solid yellow,
when the fault is over, the component switches to normal operating status.

In the scada: elipse e3 software, they have a browser where we can make requests for properties to use in scripts.
![explorador elipse](https://github.com/SCADA-LTS/Scada-LTS/assets/138117880/6f25a0f4-de16-4bac-ad11-32087a663381)

I imagine that it is possible in scada to do this search for properties to be able to apply it in scripts.

For example:

Var = (alarm name).AlarmStates ;

Something in that feeling.
![lts - alarms](https://github.com/SCADA-LTS/Scada-LTS/assets/138117880/29f4eaa7-ff86-4b4b-9728-9eb2b6e6e1e1)

**Comment by Limraj** [comment]
Hi @GabrielTec251,
thanks for creating a discussion topic. :)
Thanks @fabiodurao for taking care of this.


The first thing that comes to my mind:

1. For example, for the 'Simple Point' Component, we have the 'Warning' icon and there is a list of events associated with this point:
![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/37487cdd-8700-4c66-8de8-478abea47f54)
And:
![warning](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/1d99921d-dc73-4bf6-b237-888511f9a263)

2. In the latest version 2.7.5.4 we have improved the mechanism of informing the user about the highest level alarm, there is a sound signal and flashing:
![notify](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/589a93f8-7fba-4683-8d68-5ab4d6d377a7)

3. We have Alarm List Component:
![alarm_list_cmp](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/15b0ea13-5c27-4a88-a038-bc8a4db27192)

However, we do not have the functionality you describe... 
I think you can open that issue.

Regards,
Kamil Jarmusik

**Comment by GabrielTec251** [comment]
OK, I'll show you what I do in another software so that you guys can get a sense of how to replicate.
